src/main.zig

```diff
@@ -0,0 +1,91 @@
+const std = @import("std");
+const mem = std.mem;
+const Allocator = std.mem.Allocator;
+
+const usage =
+    \\Usage:
+    \\
+    \\  roc [options] [roc_file] [args]
+    \\  roc [command] [options]
+    \\
+    \\Commands:
+    \\
+    \\  build           Build a binary from the given .roc file, but don't run it
+    \\  test            Run all top-level `expect`s in a main module and any modules it imports
+    \\  repl            Launch the interactive Read Eval Print Loop (REPL)
+    \\  format          Format a .roc file or the .roc files contained in a directory using standard Roc formatting
+    \\  version         Print the Roc compiler’s version, which is currently built from commit 90db3b2db0, committed at 2025-01-28 18:26:51 UTC
+    \\  check           Check the code for problems, but don’t build or run it
+    \\  docs            Generate documentation for a Roc package
+    \\  glue            Generate glue code between a platform's Roc API and its host language
+    \\
+    \\General Options:
+    \\
+    \\ -h, --help       Print command-specific usage
+;
+
+pub fn fatal(comptime format: []const u8, args: anytype) noreturn {
+    std.log.err(format, args);
+    std.process.exit(1);
+}
+
+pub fn log(
+    comptime level: std.log.Level,
+    comptime format: []const u8,
+    args: anytype,
+) void {
+    const prefix = comptime level.asText();
+
+    // Print the message to stderr, silently ignoring any errors
+    std.debug.print(prefix ++ ": " ++ format ++ "\n", args);
+}
```

Comment:
Why this instead of `std.log.info` and `std.log.debug` etc?

Reply:
I was basically copying zig... and my thought was that we might have different configuration log levels. I'm probably getting a bit ahead of myself. Probably best just to remove extraneous stuff for now.

Reply:
Oh, is this how the root of the zig compiler does it? Interesting